packages/editor/src/core/components/hooks/__tests__/useNodeDropLevels.test.tsx

```diff
@@ -50,7 +50,12 @@ const SAMPLE_STATE: Value = {
 describe('useNodeDropLevels', () => {
   it('tells that cells that have siblings on the right have drop level 0 there', () => {
     const Wrapper = ({ children }) => (
-      <Provider cellPlugins={[]} lang="en" value={SAMPLE_STATE}>
+      <Provider
+        cellPlugins={[]}
+        lang="en"
+        value={SAMPLE_STATE}
+        uiTranslator={(label) => label}
```

Comment:
i don't think that this test needs to be altered

Reply:
The test is failing because `uiTranslator` is actually a required one for the `Provider`, I will change the Provider prop signature instead then.

Reply:
i think it can be optional